Tasks/AzureCLIV2/azureclitask.ts

```diff
@@ -92,6 +107,10 @@ export class azureclitask {
             }
         }
         finally {
+            if (keepAzSessionActive) {
```

Comment:
I think that this check is not needed as `stopRefreshingSession` is empty void function by default.